src/Lecture1.hs

```diff
@@ -30,6 +30,7 @@ module Lecture1
     , strSum
     , lowerAndGreater
     ) where
+import GHC.Base (VecElem(Int16ElemRep))
```

Comment:
I see the GHC warning that this import is redundant ⚠️ 
So it's safe to remove it 👍🏻 
Not sure where it's used but probably VSCode on-the-fly import addition could've been added it 🤔